openapi.yaml

```diff
@@ -599,6 +599,273 @@ paths:
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+            https://api.linode.com/v4/account/entity-transfers/123E4567-E89B-12D3-A456-426614174000
+      - lang: CLI
+        source: >
+          linode-cli account entity-transfers \
+            view 123E4567-E89B-12D3-A456-426614174000
+    delete:
+      x-linode-grant: unrestricted only
+      tags:
+      - account
+      summary: Entity Transfer Cancel
+      description: >
+        Cancels the Entity Transfer for the provided token. An email notification is sent to the account that created
+        this transfer. Transfers can not be canceled if they are expired or have been accepted.
+
+
+        This command can only be accessed by the unrestricted users of the accounts that created this transfer.
+      operationId: deleteEntityTransfer
+      x-linode-cli-action: delete
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - account:read_write
+      responses:
+        '200':
+          description: >
+            Entity Transfer canceled.
+          content:
+            application/json:
+              schema:
+                type: object
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+            -X DELETE \
+            https://api.linode.com/v4/account/entity-transfers/123E4567-E89B-12D3-A456-426614174000
+      - lang: CLI
+        source: >
+          linode-cli account entity-transfers \
+            delete 123E4567-E89B-12D3-A456-426614174000
+  /account/entity-transfers/{token}/accept:
+    x-linode-cli-command: entity-transfers
+    parameters:
+      - name: token
+        in: path
+        description: The UUID of the Entity Transfer.
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      x-linode-grant: unrestricted only
+      tags:
+      - Account
+      summary: Entity Transfer Accept
+      description: >
+        Accept an Entity Transfer for the provided token to receive the resources included in the transfer to your
+        account. A transfer must have a `pending` status in order to be accepted. Transfers can take several hours to
+        complete. Email notifications are sent to the accounts that created and accepted this transfer.
+
+
+        This command can only be accessed by the unrestricted users of the account that receives the transfer. Users
+        of the same account that created a transfer cannot accept the transfer.
+
+
+        There are several conditions that must be met in order to successfully create a transfer request:
+          1. The account accepting the transfer must have a registered payment method and must not have a past due
+          balance or other account limitations for the entities to be transferred.
```

Comment:
```suggestion
          balance, Terms of Service violations, or other account limitations for the entities to be transferred.
```

Reply:
Is this true? I'm only seeing checks for ToS violations on the Linodes subject to transfer, not generally for the receiver

Reply:
We are checking ToS tickets on accept now.